src/admin-bro.ts

```diff
@@ -226,6 +230,58 @@ class AdminBro {
     return resource
   }
 
+  resolveBabelConfigPath(): void {
+    if (typeof this.options.babelConfig === 'string') {
+      let filePath = ''
+      let config = this.options.babelConfig
+      if (config[0] === '/') {
```

Comment:
hmmm - it is a copy from bundle function. Do you have some idea how to extract this to be more dry?